### ISSUE TITLE
Include support on Windows

### ISSUE THREAD

**Issue by dylanblokhuis** [nolanderc/glsl_analyzer#14]
Great work on this project! I tried using it on windows and ran into this issue, I think the way paths are interpreted on windows is wrong and is leading to FileNotFound errors (in vscode):

```glsl
#include "global.glsl"

#define INSTANCE_MASK_OPAQUE 0x01

void main () {}
```
```
debug: hover: 14:16 file:///c%3A/Users/dylan/dev/test/shader/gbuffer.glsl
debug: hover word: 'INSTANCE_MASK_OPAQUE'
error: could not open 'file:///c%3A/Users/dylan/dev/test/shader\\global.glsl': FileNotFound
```

**Comment by nolanderc** [comment]
Right, the current implementation doesn’t support percent-encoding. I’ll try to get a fix in when I get back from work 🙂

**Comment by nolanderc** [comment]
This should be fixed in v1.1.1. If not, please reopen the issue :slightly_smiling_face:
